ADOPTERS.md

```diff
@@ -0,0 +1,6 @@
+# KubeVirt Adopters
+
+Please use this file to let us know that you are using KubeVirt by adding your company name and a description of your use case to this document.
+
+Red Hat
```

Comment:
How do we sort our adopters?

Is the formatting "Adopter : Usecase" intended? It looks nice in plain-text. Not sure how I feel about the rendered markdown, it looks nice and weird at the same time.

Reply:
I think a list with single lines would be nice? Like
`* Red Hat: Using it to run classic virtual machines`

Reply:
What about a table? Let me update.

Reply:
ok, or a table :wink: